src/components/Layout.tsx

```diff
@@ -161,27 +129,53 @@ const Layout: React.FC<IProps> = ({
                 isPageRightToLeft={isPageRightToLeft}
                 originalPagePath={pageContext.originalPath!}
               />
-              <ContentContainer>
+
+              <Flex
+                position="relative"
+                margin="0px auto"
+                minHeight="100vh"
+                flexFlow="column"
+                sx={{
+                  "@media": {
+                    minWidth: "1024px",
+                    maxWidth: "1504px",
+                  },
+                }}
+              >
                 <ZenMode>
                   <Nav path={path} />
                   {shouldShowSideNav && <SideNavMobile path={path} />}
                 </ZenMode>
                 <SkipLinkAnchor id="main-content" />
-                <MainContainer>
+                <Flex
+                  sx={{
+                    "@media (max-width: 1024px": {
+                      flexDirection: "column",
+                    },
+                  }}
```

Comment:
```suggestion
                  flexDirection={{ base: 'column', lg: 'row' }}
```
This is a normal approach 😄

Reply:
Correct. Lets do it using this approach.